x/interchainstaking/keeper/ibc_packet_handlers.go

```diff
@@ -619,7 +621,7 @@ func (k *Keeper) HandleBeginRedelegate(ctx sdk.Context, msg sdk.Msg, completion
 	k.ICQKeeper.MakeRequest(
 		ctx,
 		zone.ConnectionId,
-		zone.ChainId,
+		zone.ChainID(),
```

Comment:
I think this wants to be ID() - reason being, the query callback will be executed against the zone listed here, so and the delegation belongs to the subzone, not the rootzone.

Reply:
Same goes for receipts, balances, withdrawal / unbending records, etc. (not valsets and validators, or performance account delegations), as these will always pertain to the subzone.

icq relayer will need to be updated to split the chain ID on the separator (probably: `/` ?) and use the correct base chain id for the query itself.

Reply:
Ok - so we will want to ensure some deterministic way to get the base chainID from a subzone chainID right?

Should we just generate subzone ID as `baseID + "/something that won't conflict"?

Reply:
As discussed yesterday `baseChainID + "|" + <user-generated unique value per baseChain>`